client/pages/PostLogin/PostLoginPage.tsx

```
@@ -0,0 +1,275 @@
+          `Enter ${ph.name}${ph.hint ? ` (${ph.hint})` : ""}${ph.example ? `, e.g. ${ph.example}` : ""}:`,
+          ph.example || "",
+        );
+        if (val == null || val.trim() === "") {
+          // User cancelled or empty — abort
+          return;
+        }
+        const trimmed = val.trim();
+        if (ph.validator) {
+          try {
+            const re = new RegExp(ph.validator);
+            if (false) {
```

Biome (2.1.2)

[error] Unexpected constant condition.



(lint/correctness/noConstantCondition)